src/main/java/Send_Message/SendMessageOutputBoundary.java

```diff
@@ -0,0 +1,7 @@
+package send_message;
+
+public interface SendMessageOutputBoundary {
+    void prepareRoomView();
```

Comment:
I think you might need roomID for prepareRoomView since the messages are specific to a particular room so unique identifier of the room is required.